### ISSUE TITLE
Why not make a MySQL image base alpine

### ISSUE THREAD

**Issue by fanybook** [blacktop/docker-elasticsearch-alpine#52]
Why not make a MySQL image

:D

**Comment by blacktop** [comment]
there might already be an alpine based mysql image?

**Comment by fanybook** [comment]
No, I wonder if "Linux generic MySQL" can be successfully installed on apline

> https://wiki.alpinelinux.org/wiki/Running_glibc_programs
> https://github.com/sgerrand/alpine-pkg-glibc/releases
> https://dev.mysql.com/downloads/mysql/

**Comment by fanybook** [comment]
@blacktop Is it possible
